SCAutolib/models/file.py

```diff
@@ -360,9 +384,12 @@ def restore(self):
         #  required attributes in JSON format load() method that would be used
         #  in other then setup runtimes to restore (load from JSON) all
         #  attributes of this object
-        self.clean()
+
         if self._backup_original:
             copy2(self._backup_original, self._conf_file)
+        else:
+            self.clean()
+        self._changed = False
```

Comment:
No restart of anything is needed after restoring config file?

Reply:
It is needed, but I would prefer to leave this to the caller (similar to the `save` method) for better testability.